src/cordova/apple/OutlineTun2Socks/Package.swift

```diff
@@ -0,0 +1,32 @@
+// swift-tools-version: 5.7
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "OutlineTun2Socks",
+    products: [
+        .library(
+            name: "OutlineTun2Socks",
+            targets: ["Tun2socks", "OutlineDependencies"]),
+    ],
+    dependencies: [
+      .package(url: "https://github.com/CocoaLumberjack/CocoaLumberjack.git", from: "3.8.0"),
+      .package(url: "https://github.com/getsentry/sentry-cocoa", from: "7.31.5"),
+    ],
+    targets: [
+        .target(
+            name: "OutlineDependencies",
```

Comment:
This target seems odd to me. Why do we need it?

Let's go step by step here. For now, let's hold on depending on Sentry and CocoaLumberjackSwift here. The app code must still depend on them directly anyway, since that's what uses them. So we are not gaining much by having them here and it makes the dependency chain harder to reason about.

So Step 1: Tun2socks only

Step 2 two would be to add some code to this package. Perhaps the PacketTunnelProvider? Then we could have it pull whatever dependencies it needs (and only those dependencies).

Eventually we will pull all our code from the app, and we can remove Sentry and CocoaLumberjack from there once no more code there depends on it.